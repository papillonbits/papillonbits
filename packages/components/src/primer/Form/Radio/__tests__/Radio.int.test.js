import { regular, inactive } from './Radio.int.story'

describe('<Radio />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })

    test('must match inactive()', () => {
      expect(global.renderToJSON(inactive())).toMatchSnapshot()
    })
  })
})
