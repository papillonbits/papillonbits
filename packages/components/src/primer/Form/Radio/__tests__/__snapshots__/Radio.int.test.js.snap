// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Radio /> Render must match inactive() 1`] = `
<div>
  <div
    class="m-0 form-checkbox radio-inactive"
  >
    <label
      for="checkbox"
    >
      <input
        name="fantastic-name"
        type="radio"
      />
      White
    </label>
  </div>
  <div
    class="m-0 form-checkbox radio-inactive"
  >
    <label
      for="checkbox"
    >
      <input
        checked=""
        name="fantastic-name"
        type="radio"
      />
      Black
    </label>
  </div>
</div>
`;

exports[`<Radio /> Render must match regular() 1`] = `
<div>
  <div
    class="m-0 form-checkbox"
  >
    <label
      for="checkbox"
    >
      <input
        name="fantastic-name"
        type="radio"
      />
      White
    </label>
  </div>
  <div
    class="m-0 form-checkbox"
  >
    <label
      for="checkbox"
    >
      <input
        checked=""
        name="fantastic-name"
        type="radio"
      />
      Black
    </label>
  </div>
</div>
`;
